community/modules/scheduler/schedmd-slurm-gcp-v6-controller/login.tf

```diff
@@ -59,26 +59,27 @@ module "slurm_login_template" {
 
 # INSTANCE
 module "slurm_login_instance" {
-  source   = "github.com/GoogleCloudPlatform/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_login_instance?ref=6.4.3&depth=1"
+  source   = "github.com/GoogleCloudPlatform/slurm-gcp.git//terraform/slurm_cluster/modules/_slurm_instance?ref=6.4.3&depth=1"
   for_each = { for x in var.login_nodes : x.name_prefix => x }
 
+  access_config       = each.value.access_config
+  add_hostname_suffix = true
+  hostname            = "${local.slurm_cluster_name}-login-${each.key}"
+  slurm_instance_role = "login"
+
   project_id         = var.project_id
   slurm_cluster_name = local.slurm_cluster_name
 
-  enable_public_ip = each.value.enable_public_ip
   instance_template = (
     each.value.instance_template != null && each.value.instance_template != ""
     ? each.value.instance_template
     : module.slurm_login_template[each.key].self_link
   )
-  network_tier  = each.value.network_tier
+  labels        = merge(each.value.labels, local.files_cs_labels)
   num_instances = each.value.num_instances
 
   region     = each.value.region
```

Comment:
There is a slight difference here. Before, due to using slurm login node module, the login node would use region of subnetwork and only fall back to var.region if region was not knowable from subnetwork. I am trying to think through all ramifications.

In our wrapper modules we always require subnetwork_self_link and region is optional. I am wondering if login node should take region at all. Would you ever deploy into a region different than the subnetwork.

Reply:
We had offline discussion here. I am going to leave it open for now so I remember to return to it.

Reply:
While I agree with importance of resolving "explicit region/zone vs deduced from mandatory subnetwork".  I think this PR doesn't make it any worse, since login node also has `var.zone`(before and after the change) that is passed through.

I propose to handle this issue separately.